dino.icu.yaml

```
@@ -440,6 +440,10 @@ xn--rl8h:
 yc:
   type: NS
   value: ns.youngchief.xyz.
+
+yates: 
+  type: CNAME
+  value: peteryates.net
```

CNAME records require a trailing dot
```suggestion
  value: peteryates.net.
```